<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1055</width>
    <height>820</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Mapa de Calor</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btn_subirimg">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>40</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Subir Imagen JPG/PNG</string>
    </property>
   </widget>
   <widget class="QLabel" name="Label_img">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>90</y>
      <width>871</width>
      <height>491</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_csv">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>40</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Subir Archivo .csv</string>
    </property>
   </widget>
   <widget class="QToolButton" name="Btn_recortar">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>630</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Recortar Esquema</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Btn_save">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>630</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Guardar Esquema</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_subirpdf">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>40</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Subir PDF</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>40</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Subir DWG</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1055</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
